nginx.conf

```diff
@@ -1,29 +1,28 @@
 server {
-	listen 80 default_server;
+  listen 80 default_server;
 
-	gzip on;
-	gzip_min_length 1000;
-	gzip_types text/plain text/xml application/javascript text/css;
+  gzip on;
+  gzip_min_length 1000;
+  gzip_types text/plain text/xml application/javascript text/css;
 
-	root /app;
+  root /app;
 
-	# normal routes
-	# serve given url and default to index.html if not found
-	# e.g. /, /user and /foo/bar will return index.html
-	location / {
-		add_header Cache-Control "no-store";
-		add_header Content-Security-Policy "frame-ancestors 'self'";
-		add_header X-Content-Type-Options "nosniff";
-		add_header X-Frame-Options "SAMEORIGIN";
```

Comment:
This was removed. It doesn't seem to support anything other than deny or same origin in the current incarnation. content security policy seems to be supported and in effect for browsers. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options#allow-from_origin

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors#browser_compatibility

Open to feedback on this.